gradle.properties

```diff
@@ -43,3 +44,6 @@ kotlin.native.ignoreDisabledTargets=true
 
 # Site deneration
 jekyll_version=4.0
+
+# JS IR baceknd sometimes crashes with out-of-memory
```

Comment:
Please create an issue for that and link it here so we can keep track of it and remove this option as soon as it's fixed

Reply:
Done